pkg/graveler/settings/manager.go

```diff
@@ -83,6 +83,19 @@ func (m *Manager) Save(ctx context.Context, repositoryID graveler.RepositoryID,
 	return nil
 }
 
+func (m *Manager) GetLatest(ctx context.Context, repositoryID graveler.RepositoryID, key string, settingTemplate proto.Message) (proto.Message, error) {
+	messageBytes, err := m.getFromStore(ctx, repositoryID, key)
+	if err != nil {
+		return nil, err
+	}
+	setting := proto.Clone(settingTemplate)
```

Comment:
Is this a common pattern? I would expect to use the given object all the way, fill it with the data and return just the error.